### PR TITLE
fix(docs): hot-reload styles, fix trailing slashes

### PR DESCRIPTION
**Pull request by redallen**
Towards #2343

### CODE REVIEW

**Comment by mattnolting** [comment]
Fixes live reload, @mcoker can we merge asap?

File: gatsby-browser.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
+// Patternfly CSS
+import './dist/patternfly.css';
+// Utilities
+import './dist/patternfly-addons.css';
+// Experimental components
+import './dist/components/DataToolbar/data-toolbar.css';
+import './dist/components/Divider/divider.css';
+import './dist/components/Drawer/drawer.css';
+import './dist/components/OverflowMenu/overflow-menu.css';
+import './dist/components/Spinner/spinner.css';
```
Comment (mcoker) [inline gatsby-browser.js]: Just curious - is this something we'll need to maintain as we add new experimental features?

File: package.json [hunk 2]
```diff
@@ -27,7 +28,7 @@
     "precommit": "lint-staged",
     "sassdoc": "sassdoc ./src/patternfly/**/*.scss --parse > sassdoc/data.json",
     "serve": "gatsby serve",
-    "start": "npm run develop",
+    "start": "concurrently \"gulp start\" \"gatsby develop\"",
```
Comment (mcoker) [inline package.json]: ah, folks also use `npm run dev` and `npm run develop` to start the dev server. The main README references `npm run dev` as the way to start the server - https://github.com/patternfly/patternfly-next#development

Can we update those to match? Or maybe consolidate/remove the extras and tell everyone? Also is there a way to replicate `dev:expose` where the server binds to your network IP, and not just localhost/127.0.0.1?

**Comment by mcoker** [comment]
@redallen I pulled this branch down locally, removed `node_modules`, ran `npm install`, and started the server with `npm run start`, and hot reloading styles doesn't seem to work. The browser refreshes, but I don't see changes I've made to the SCSS. Manually reloading the page doesn't show me changes, either. I also noticed that full-page examples throw an error locally, and they aren't working on the PR preview URL, either. They don't error there, they just don't go anywhere when you click on the link.

**Comment by redallen** [comment]
@mcoker the fullscreen pages got broken when I added them for React yesterday. I'll go back in and fix that.

As for the hot reloading styles, they worked for @mattnolting and I yesterday and it's rather complex to get them working _quickly_ and _correctly_. We use a glob importer and do intermediate steps in our build like injecting `@import "../../patternfly-imports";` at the top of each file. I'll spend the morning playing around with it. I'll ditch gulp-sass, use the new node-sass and start poking around at how our `dist` gets created.

File: gatsby-browser.js (hunk 1 above)
Comment (redallen) [reply gatsby-browser.js]: Yep! Consumers have to do this as well.

File: src/patternfly/components/_all.scss [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+@import "./Card/card.scss";
+@import "./Check/check.scss";
+@import "./Chip/chip.scss";
+@import "./ChipGroup/chip-group.scss";
+@import "./ClipboardCopy/clipboard-copy.scss";
+@import "./Content/content.scss";
+@import "./ContextSelector/context-selector.scss";
+@import "./DataList/data-list.scss";
+
+// @import "./DataToolbar/data-toolbar.scss";
```
Comment (redallen) [inline src/patternfly/components/_all.scss]: @mcoker You'll also have to track this guy for the experimental features.

**Pull request merged**